src/LocalMustacheTemplateController.js

```diff
@@ -0,0 +1,19 @@
+const { MockKubeResourceMeta } = require('@razee/razeedeploy-core');
```

Comment:
lets move this file to either `test/` directory, or a new directory for local rendering